libnavigation-ui/src/main/java/com/mapbox/navigation/ui/instruction/turnlane/TurnLaneViewHolder.java

```diff
@@ -5,6 +5,7 @@
 import androidx.recyclerview.widget.RecyclerView;
 
 import com.mapbox.libnavigation.ui.R;
+import com.mapbox.navigation.ui.instruction.turnlane.internal.TurnLaneView;
```

Comment:
Typo, it should be `com.mapbox.navigation.ui.internal.instruction.turnlane.TurnLaneView`